javascript/src/URDFLoader.js

```diff
@@ -105,7 +103,7 @@ class URDFLoader {
         const packages = options.packages || '';
         const loadMeshCb = options.loadMeshCb || this.defaultMeshLoader.bind(this);
         const workingPath = options.workingPath || '';
-        const parseVisual = options.parseVisual || false;
+        const parseVisual = ('parseVisual' in options) ? options.parseVisual : true;
```

Comment:
I don't meant to get too picky but it might be best to follow the default pattern used by the other options:

```js
const parseVisual = options.parseVisual || true;
```

If `parseVisual` isn't in the object definition then it will be `undefined` so `parseVisual` will be set to `true`.

Reply:
Am I missing something? If it is in the object definition and is given as `false` it'll fallback to `true`?
An `or true` could never be `false`.

Reply:
Oh no I'm sorry you're right -- it's definitely more robust, too. I'll look into changing the rest of the options to the inline conditional in a subsequent commit.

Ha, maybe it's too early for PRs! Thanks.